src/Campaigns/DataTransferObjects/CampaignDetailsData.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Give\Campaigns\DataTransferObjects;
+
+use Give\Campaigns\Models\Campaign;
+use Give\Framework\Support\Contracts\Arrayable;
+
+/**
+ * @unreleased
+ */
+class CampaignDetailsData implements Arrayable
+{
+    /**
+     * @var Campaign
+     */
+    private $campaign;
+
+    /**
+     * @unreleased
+     */
+    public function __construct(Campaign $campaign)
+    {
+        $this->campaign = $campaign;
+    }
+
+    /**
+     * @unreleased
+     */
+    public function toArray(): array
+    {
+        return array_merge(
+            $this->campaign->toArray(),
+            [
+                'defaultFormId' => $this->campaign->defaultForm()->id,
```

Comment:
I think we don't need this DTO since the campaign model will have the default form ID available when this PR is merged: https://github.com/impress-org/givewp/pull/7616

Reply:
Good point. I'm looking at #7616 now.